static/toggles.js

```diff
@@ -27,53 +27,88 @@ var _ = require('underscore');
 var $ = require('jquery');
 var EventEmitter = require('events');
 
-function get(buttons) {
-    var result = {};
-    _.each(buttons, function (a) {
-        var obj = $(a);
-        result[obj.data().bind] = obj.hasClass("active");
-    });
-    return result;
-}
 
-function Toggles(root, state) {
+function Togglesv2(root, state) {
```

Comment:
Was there a name clash with something, or just reflecting the changes with a new name? Just interested :)

Reply:
Reflecting the changes with a new name. It also allowed me to have both versions available while implementing it